fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/adbrun.py

```diff
@@ -0,0 +1,17 @@
+import logging
+import os
+
+logging.getLogger(__name__).addHandler(logging.NullHandler())
+
+
+class ADBrun(object):
+    binary = "adbrun"
+    logger = logging.getLogger()
+
+    def launch(self):
+        # First close sim if any then launch
+        os.system(
+            "~/Library/Android/sdk/platform-tools/adb devices | grep emulator | cut -f1 | while read line; do ~/Library/Android/sdk/platform-tools/adb -s $line emu kill; done"
+        )
+        # Then launch sim
+        os.system("sh launchSimScript.sh")
```

Comment:
Ideally we should be re-using the existing helper files rather than copying them again. Maybe a follow-up.

Reply:
Fair, I don't think I even use this adb run file.